backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/paymentallocation/service/PaymentAllocationBuilder.java

```diff
@@ -40,6 +67,9 @@
 		DO_NOTHING, WRITE_OFF, DISCOUNT
 	}
 
+	private final MoneyService moneyService = SpringContextHolder.instance.getBean(MoneyService.class);
+	private final ICurrencyBL currencyBL = Services.get(ICurrencyBL.class);
```

Comment:
pls move those near the other services declared in this class... see line 78

Reply:
done